Docs/source/pages/commands.md

```diff
@@ -2755,7 +2755,7 @@ Gets the value of the static field or property.
 | ------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------- |
 | componentName | string | Yes      | The name of the component which has the static field or property to be retrieved.                       |
 | propertyName  | string | Yes      | The name of the static field or property to be retrieved.                                               |
-| assembly      | string | Yes      | The name of the assembly the component belongs to.                                                      |
+| assemblyName  | string | Yes      | The name of the assembly the component belongs to.                                                      |
```

Comment:
For `SetStaticProperty` the description is different, you may use the same for the first part to be consistent in the way we pass information
![image](https://user-images.githubusercontent.com/62327834/230910253-4e273e76-9495-43fe-ab36-b21d374146f1.png)